engine/src/test/scala/cromwell/engine/backend/jes/JesBackendSpec.scala

```diff
@@ -52,18 +52,19 @@ class JesBackendSpec extends FlatSpec with Matchers with Mockito with BeforeAndA
       gcsFilesystemAuth = clientSecrets)
   }
 
-  "executionResult" should "handle Failure Status" in {
+  ignore should "handle Failure Status" in {
+  //"executionResult" should "handle Failure Status" in {
```

Comment:
To avoid losing the information that it's `executionResult` which `should` be doing something